tests/Testcases/trivial/test.tcl

```diff
@@ -0,0 +1,16 @@
+create_design test
+set_top_module top
+set_macro P1=10  P2=20
+add_include_path ./ inc/
+add_design_file test.v bottom.v -SV_2012
+synth
+packing
+place
+route
+sta
+power
+bitstream
+puts "done!"
```

Comment:
sync is missing here

Reply:
Added temporarily until you provide a solution that does not require "sync".